prefect_aws/workers/glue_job_worker.py

```diff
@@ -0,0 +1,156 @@
+import logging
+import time
+from typing import Any, Optional
+
+import anyio
+from prefect.server.schemas.core import FlowRun
+from prefect.utilities.asyncutils import run_sync_in_worker_thread
+from prefect.workers.base import (
+    BaseJobConfiguration,
+    BaseVariables,
+    BaseWorker,
+    BaseWorkerResult,
+)
+from pydantic import Field
+
+from prefect_aws import AwsCredentials
+
+_GlueJobClient = Any
+
+
+class GlueJobWorkerConfiguration(BaseJobConfiguration):
+    """
+    Job configuration for a Glue Job.
+    """
+
+    job_name: str = Field(
+        ...,
+        title="AWS Glue Job Name",
+        description="The name of the job definition to use.",
+    )
+    arguments: Optional[dict] = Field(
+        default=None,
+        title="AWS Glue Job Arguments",
+        description="The job arguments associated with this run.",
+    )
```

Comment:
I like how flexible this field is, but it'd be nice to have some stronger typing while maintaining flexibility. Introducing a variables class that contains some values that will often vary between deployments and creating a default template can help with that. You can checkout the ECS worker's [variables class](https://github.com/PrefectHQ/prefect-aws/blob/679fad1c6a55a30240212302d0f579d42dbab445/prefect_aws/workers/ecs_worker.py#L344) as an example.

Reply:
Thanks, fix it. 
Does this match what you had in mind?
https://github.com/PrefectHQ/prefect-aws/pull/308/commits/e71b9476ac0b8b531c08c1838a22694e7402e27b